### PR TITLE
Add Secret Santa Trophies Of 2015

### PR DESCRIPTION
**Pull request by McFanta**

### CODE REVIEW

File: en/wiki/trophies.html [hunk 1]
```diff
@@ -1,5 +1,4 @@
-<html lang="en">
-  <head>
```
Comment (Eudaldca) [inline en/wiki/trophies.html]: Don't delete head if you want it to work :stuck_out_tongue:

**Comment by McFanta** [comment]
;P! I thought Matic were working on a new one :P!

File: en/wiki/trophies.html [hunk 2]
```diff
@@ -331,5 +356,6 @@ <h2 id="july_is_the_new_december">July is the new December</h2>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>
 
-  </body>
-</html>
+  
+
+</body></html>
```
Comment (Eudaldca) [inline en/wiki/trophies.html]: Keep the indent as it was please

**Comment by matic0basle** [comment]
I was working on a new one with a team but that project isn't currently being worked on. I am working on IRL projects and others are busy as well.

**Comment by McFanta** [comment]
At least could you just add for fun? Just to bring some activity here ;P

**Comment by matic0basle** [comment]
Since this isn't being updated, I won't merge anything.

**Comment by McFanta** [comment]
Oh okay.

**Pull request closed without merging**